lib/core/event_builder/index.js

```diff
@@ -162,11 +163,17 @@ module.exports = {
    * @param  {Object} attributes    Object representing user attributes and values which need to be recorded
    * @return {Object}               Params to be used in impression event logging endpoint call
    */
-  createImpressionEventParams: function(configObj, experimentKey, variationId, userId, attributes) {
-    var params = getCommonParams(configObj, userId, attributes);
+  getImpressionEvent: function(configObj, experimentKey, variationId, userId, attributes) {
```

Comment:
since we are already refactoring, we should make this method take in a `options` obj instead of having a super long method signature